lib/content-services/src/lib/permission-manager/components/user-icon-column/user-icon-column.component.scss

```diff
@@ -24,20 +24,15 @@
         font-size: var(--theme-title-font-size);
     }
 
-    &-people-select-icon {
-        margin: 0 !important;
+    &-people-select-icon.adf-datatable-selected {
+        margin: 2px -2px 0 0 !important;
+        height: 100%;
+        width: 100%;
 
         svg {
             fill: var(--theme-accent-color);
             width: 40px;
             height: 40px;
         }
     }
-
-    &-datatable-selected.mat-icon {
-        height: 100%;
-        width: 100%;
-        margin-left: -2px;
```

Comment:
This styles are not always used with adf-people-select-icon class. For example here: 
![Screenshot 2024-01-24 at 14 57 30](https://github.com/Alfresco/alfresco-ng2-components/assets/84377976/cfbfcb9e-7960-4db5-8565-33fe48e1624f)
So now magin is not applied and icon has wrong alignment 
![Screenshot 2024-01-24 at 14 58 51](https://github.com/Alfresco/alfresco-ng2-components/assets/84377976/f2217454-ad24-4d07-8f92-4bebc7add14c)